doc/conf.py

```diff
@@ -153,12 +153,10 @@
     'bootstrap_version': "3",  # default
     'navbar_links': [
         ("Get started", "getting_started"),
-        ("Tutorials", "tutorials"),
-        ("Examples", "auto_examples/index"),
+        ("Documentation", "documentation"),
         ("API", "python_reference"),
-        ("Manual", "manual/index"),
-        ("Contribute", "contribute_to_mne"),
-        ("FAQ", "faq"),
+        ("Examples", "auto_examples/index"),
```

Comment:
I am wondering if this could even be called "code examples" to emphasize that this is really about exposing ideas outside of the context of a full analysis.